mapper/cumulocity/c8y_translator_lib/src/json.rs

```diff
@@ -219,4 +240,38 @@ mod tests {
         );
         }
     }
+
+    #[test_case(
+    "{\"temperature\": 23.0, \"pressure\": 220.0}\"",
+    "child1"
+    ;"input thin-edge json")]
```

Comment:
```suggestion
    ;"child device thin-edge json translation")]
```
To better reflect what the test is about;